app/lib/lti_advantage/params.rb

```diff
@@ -0,0 +1,58 @@
+module LtiAdvantage
+  # This is for extracting data from the lti jwt in more human-readable ways
+  class Params
+    attr_reader :token
+
+    def initialize(lti_token)
+      @token = lti_token
+    end
+
+    def launch_context
+      # This is an array, I'm not sure what it means to have more than one
+      # value. In courses and accounts there's only one value
+      context = context_data["type"][0]
```

Comment:
context is optional. In Canvas it is always there and I would guess most people provide it but it could be null. We might want to check for that.

Reply:
yeah it's probably a good idea to have more safe navigation in here, I'll add some of that